src/style.css

```diff
@@ -1 +1,135 @@
-/* add styles here */
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir';
+  font-weight: 300;
+  font-style: normal;
+  src: url('../src/fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir';
+  font-weight: 600;
+  font-style: normal;
+  src: url('../src/fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+form {
+  display: block;
+  unicode-bidi: isolate;
+}
+
+body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir', sans-serif;
+  font-size: 16px;
+}
+
+label {
+  cursor: default;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-form_big {
+  width: 100%;
+  height: 70px;
+}
+
+.search-form_small {
+  width: 100%;
+  height: 42px;
+}
+
+.search-form__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+/* stylelint-disable-next-line selector-pseudo-element-colon-notation */
+.search-form__label:after {
+  position: absolute;
+  content: '';
+  display: block;
+}
+
+/* stylelint-disable-next-line selector-pseudo-element-colon-notation */
+.search-form__label_big:after {
+  background-image: url('../src/images/Search.svg');
+  background-repeat: no-repeat;
+  background-size: contain;
```

Comment:
This styles should not be doublicated, use .search-form__label selector to display the styles